pkg/api/api.go

```diff
@@ -129,8 +128,9 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 		Start(
 			ctx,
 			"event-ingestion",
+			trace.WithNewRoot(),
```

Comment:
we can just force a new root this way without needing to utilize a separate context.

Reply:
@jpwilliams are there cases we don't want it to be a root span?
if the source is from the event API, I think it's fine they're all root spans atm?